source/javascript/src/collection-edit.js

```diff
@@ -72,12 +78,16 @@ function populateTasks (collection) {
   }
   const tasks = collection.tasks
   const taskList = document.getElementById('myUL')
-  tasks.forEach((task, index) => {
-    const logItem = createLogItem(task)
-    const li = document.createElement('li')
-    li.appendChild(logItem)
-    taskList.appendChild(li)
-  })
+  try {
+    tasks.forEach((task, index) => {
+      const logItem = createLogItem(task)
+      const li = document.createElement('li')
+      li.appendChild(logItem)
+      taskList.appendChild(li)
+    })
+  } catch (err) {
+    console.log('collection has no tasks')
```

Comment:
As a stretch goal, we may want to look into adding a task for error handling (displaying something to the user when there are no collections). @yuzilyu  what do you think?

Reply:
The error handling here isn't actually needed, I added it for debugging and forgot to get rid of it. I think having some watermark that indicates the page is empty would be nice, but definitely a stretch goal.

Reply:
I think it's okay to show an empty page, just make the button to have the text saying "Add your first collection" or something to implicitly tell the user that there isn't any collections yet and they need to add one